turbinia/workers/binary_extractor.py

```diff
@@ -41,21 +43,44 @@ def run(self, evidence, result):
     binary_extraction_evidence = BinaryExtraction()
 
     binary_extraction_evidence.local_path = self.output_dir
-    image_export_log = os.path.join(
-        self.output_dir, '{0:s}.log'.format(self.id))
+    image_export_log = os.path.join(self.output_dir, 'binary_extraction.log')
+
+    binary_extraction_dir = os.path.join(self.output_dir, 'extracted_binaries')
 
     cmd = [
-        'image_export.py', '--signatures', 'elf,exe_mz', '--logfile',
-        image_export_log
+        'image_export.py', '--partitions', 'all', '--no_vss', '--signatures',
+        'elf,exe_mz', '--logfile', image_export_log
     ]
     if config.DEBUG_TASKS:
       cmd.append('-d')
-    cmd.extend(['-w', self.output_dir, evidence.local_path])
+    cmd.extend(['-w', binary_extraction_dir, evidence.local_path])
 
     result.log('Running image_export as [{0:s}]'.format(' '.join(cmd)))
 
     self.execute(
         cmd, result, log_files=[image_export_log],
         new_evidence=[binary_extraction_evidence], close=True)
 
+    json_path = os.path.join(
+        self.output_dir, 'extracted_binaries', 'hashes.json')
+
+    with open(json_path) as json_file:
+      hashes = json.load(json_file)
+
+    files = next(os.walk(binary_extraction_dir))[2]
+    hash_cnt = len(hashes)
+    binary_cnt = len(files) - 1
```

Comment:
Why the -1?  Is there a directory or something?  Maybe we can add a test that generates files a couple directories deep to make sure this is all working correctly?

Reply:
binary_extraction_dir besides the extracted binaries, contains hashes.json file, that why you see -1 here. I'll add tests to make sure we have this covered.